test/core/templaters/dbt_test.py

```diff
@@ -120,14 +120,82 @@ def test__templater_dbt_templating_test_lex(in_dbt_project_dir, dbt_templater):
 )
 @pytest.mark.dbt
 def test__templated_sections_do_not_raise_lint_error(in_dbt_project_dir, fname):  # noqa
-    """Test that the dbt test has only a new line lint error."""
+    """Test that templated sections in dbt model does not raise a linting error."""
     lntr = Linter(config=FluffConfig(configs=DBT_FLUFF_CONFIG))
     lnt = lntr.lint_path(path="models/my_new_project/" + fname)
     violations = lnt.check_tuples()
     print(violations)
     assert len(violations) == 0
 
 
+@pytest.mark.parametrize(
+    "fname",
+    [
+        "single_trailing_newline.sql",
```

Comment:
I think this fixture should be added to https://github.com/sqlfluff/sqlfluff/pull/868/files#diff-5d3be0ba02e85767f3906e01bdb3611ce1c9defc129ba33dde3c2ecd6e6727e0R118 rather than the code being copied

Reply:
I've simplified the tests to minimize repeated code and use fixtures.